### PR TITLE
Uther Beard

### PR DESCRIPTION
**Pull request by ercarp**
## Changelog:
- Made a custom beard to be used by Uther.

<details><summary>Click to expand</summary>

image

</details>

# How to test:
- Look through the code and make sure everything is working correctly and that there are no errors or unnecessary changes.

### CODE REVIEW

File: gfx/models/portraits/attachments/male_beards/western/male_beard_western_09_wc/male_beard_western_09_wc.asset [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	}
+}
+
+entity = {
+	name = "male_beard_western_09_wc_entity"
+	pdxmesh = "male_beard_western_09_wc_mesh"
+
+	attribute = { name = "bs_fat_1"					blend_shape = "male_beard_western_09_bs_fat" }
+	attribute = { name = "bs_old"					blend_shape = "male_beard_western_09_bs_old" }
+}
```
Comment (zumbak04) [inline gfx/models/portraits/attachments/male_beards/western/male_beard_western_09_wc/male_beard_western_09_wc.asset:25]: What is changed here compared to the original one?

File: common/genes/03_genes_accessories_beards.txt [hunk 2]
```diff
@@ -283,5 +283,14 @@ accessory_genes = {
 				1 = empty
 			}
 		}
+		uther_beard = {
```
Comment (zumbak04) [inline common/genes/03_genes_accessories_beards.txt:286]: The question, do you plan to force this beard on Uther using a portrait modifier?

File: gfx/models/portraits/attachments/male_beards/western/male_beard_western_09_wc/male_beard_western_09_wc.asset (hunk 1 above)
Comment (ercarp) [reply gfx/models/portraits/attachments/male_beards/western/male_beard_western_09_wc/male_beard_western_09_wc.asset:25]: > 
> 
> What is changed here compared to the original one?

I changed the name from `male_beard_western_09` to `male_beard_western_09_wc` for the necessary files. Although the only one that really needed to be changed is `male_beard_western_09_normal_wc.dds`, so if you want to change the others back, feel free to do so.

File: common/genes/03_genes_accessories_beards.txt (hunk 2 above)
Comment (ercarp) [reply common/genes/03_genes_accessories_beards.txt:286]: > 
> 
> The question, do you plan to force this beard on Uther using a portrait modifier?

Yes, I will make a portrait modifier similar to what Lothar has so Uther's beard changes to this one when he's 40 or so.

File: gfx/models/portraits/attachments/male_beards/western/male_beard_western_09_wc/male_beard_western_09_wc.asset (hunk 1 above)
Comment (zumbak04) [reply gfx/models/portraits/attachments/male_beards/western/male_beard_western_09_wc/male_beard_western_09_wc.asset]: Removed non needed files in 6a9a11084a216ca412cd293bfe3891259210b85c.

**Review by zumbak04 — APPROVED** [review]
LGTM! Check if this works for you before the merge.

**Pull request merged**